### PR TITLE
Fix JENKINS-19811. Change JSCH version to 0.1.51.

### PR DESCRIPTION
**Pull request by ericcitaire**
Seems to work just fine (tested on a fresh install of Jenkins 1.595). I hope a release can be available soon for this fix.

Thanks,
Eric.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@ Based on the cool scp plugin.</description>
 	<parent>
 		<groupId>org.jenkins-ci.plugins</groupId>
 		<artifactId>plugin</artifactId>
-		<version>1.424</version>
+		<version>1.596</version>
```
Comment (oleg-nenashev) [inline pom.xml]: Is there any real need to change the core dependency?

Comment (ericcitaire) [inline pom.xml]: Not at all, I figured it couldn't hurt to update this version as well. But maybe I was wrong.
Does it introduce incompatibilities with earlier versions of Jenkins?
Sorry, I'm not familiar with Jenkins plugin development.

Comment (ericcitaire) [inline pom.xml]: I rolled-back this modification.

**Comment by ericcitaire** [comment]
Can someone review the last commit please?
Thanks.

**Comment by jglick** [comment]
Any particular reason you closed this?

**Comment by ericcitaire** [comment]
I made another PR with only one commit, to be more concise. And Jenkins was complaining about by second commit. Don't know exactly why...

**Comment by jglick** [comment]
#11 I suppose.

Personally I do not advocate rebasing just to create an artificially simplified history, but some people like it. In any event, GitHub lets you force-push to a branch and updates the PR accordingly, so it is not actually necessary to create a separate PR.

**Pull request closed without merging**